src/krux/firmware.py

```diff
@@ -250,6 +235,21 @@ def status_text(text, highlight_prefix=""):
         display.flash_text(t("Bad signature"), theme.error_color)
         return False
 
+    status_text(
+        t("New firmware detected.")
+        + "\n\n"
+        + "SHA256:\n\n"
+        + binascii.hexlify(firmware_hash).decode()
+        + "\n\n\n"
+        + t("Install?"),
+        ":",
+    )
+    inp.buttons_active = True
+    if inp.wait_for_button() in (BUTTON_PAGE, BUTTON_PAGE_PREV):
+        display.clear()
+        inp.wait_for_release()  # Wait for button release loading inputs on context
+        return False
+
```

Comment:
The general coverage also went down 3 lines. This may be due the test when user negates installing the firmware is missing mocks for a valid file / signature

Reply:
Yes, I'll fix coverage before setting it as ready to review